### ISSUE TITLE
Is there a way to integrate themes into Flask GoogleMaps?

### ISSUE THREAD

**Issue by aswinckr** [rochacbruno/Flask-GoogleMaps#51]
I would like to use themes from https://mapstyle.withgoogle.com/ . Is there a way to get this integrated into this plugin?

**Comment by rochacbruno** [comment]
interesting, never heard about themes, looks like it is just a json + style args in the url. looks easy to implement. (but not supported yet).

![screenshot_2017-05-28_10-49-50](https://cloud.githubusercontent.com/assets/458654/26529276/83de8be8-4393-11e7-89a5-b53aa68655a3.png)

**Comment by aswinckr** [comment]
Yep looks fairly simple :) would be great to have the feature in there. 😀

**Comment by step21** [comment]
I could work on it I think. I just have to think about where to add this. From what I understand, for each part of the map (road, etc) a colour is defined in the url, which would make the url very long. Is that ok?
Similarly -

* Do you want to just allow each colour to be set separately, leaving the color choice to the user, or do you want to copy the 'themes' from that site?
* Similarly, do you also want to offer other options that are used there such as density?

**Comment by ar-anvd** [comment]
+1 for this
